test/lib/assertHelpers.js

```diff
@@ -36,13 +36,13 @@ var AssertHelpers = {
             });
 
             it('report', function() {
-                assert(checker.checkString(options.input).getErrorCount() === options.errors);
+                expect(checker.checkString(options.input)).to.have.error.count.equal(options.errors);
```

Comment:
@markelog It works like: `expect(${ACTUAL}).to.${ACTION}(${EXPECTED})`. Not sure we can do it better ;-)